app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -517,8 +517,10 @@ public void applyTextFormat(int textFormatId) {
                 .appendTextActionsToBar(_textActionsBar);
         if (_textActionsBar.getChildCount() == 0) {
             _textActionsBar.setVisibility(View.GONE);
+            ((HorizontalScrollView) _textActionsBar.getParent()).setVisibility(View.GONE);
         } else {
             _textActionsBar.setVisibility(View.VISIBLE);
+            ((HorizontalScrollView) _textActionsBar.getParent()).setVisibility(View.VISIBLE);
```

Comment:
I am not able to get the textactions bar to hide. There is always a band visible. Any suggestions here would be appreciated.